rhombus/scribblings/ref-list.scrbl

```diff
@@ -54,15 +60,17 @@ it supplies its elements in order.
 
 @doc(
   ~nonterminal:
-    list_expr: block expr
+    listable_expr: block expr
   fun List(v :: Any, ...) :: List
   expr.macro '#%brackets [$expr_or_splice, ...]'
   repet.macro '#%brackets [$repet_or_splice, ...]'
-
+  expr.macro 'List [$expr_or_splice, ...]'
+  repet.macro 'List [$repet_or_splice, ...]'
```

Comment:
I missed this one; the space should be removed.

Reply:
HF